src/theme/NotFound.js

```diff
@@ -29,4 +30,30 @@ function NotFound() {
   );
 }
 
+/**
+ * @param {string} path
+ * @returns {string}
+ */
+ function getLanguageByPath(path) {
+  if (path.startsWith('/python'))
+    return 'Python';
+  if (path.startsWith('/java'))
+    return 'Java';
+  if (path.startsWith('/dotnet'))
+    return '.NET';
+  return 'Node.js'
+}
+
+/**
+ * @param {string} path
+ * @returns {string}
+ */
+function generateGitHubIssueLink(path) {
+  const title = `[Docs]: Page not found ${path}`;
+  const body = `Hello!
+
+I was navigating to the following URL: https://playwright.dev/${path} and it resulted in a 404 error.`;
+  return `https://github.com/microsoft/playwright/issues/new?title=${encodeURIComponent(title)}&body=${encodeURIComponent(body)}`;
```

Comment:
URLSearchParams are for some reason leading to a weird error (`TypeError: chalk_1.default.bold is not a function`). The workaround works!